modules/vaos/spec/services/v2/appointment_service_spec.rb

```diff
@@ -628,6 +628,25 @@
           expect(SchemaContract::Validation.last.status).to eq('success')
         end
       end
+
+      context 'includes travel claims' do
+        it 'returns a list of appointments with travel claim information attached' do
+          Flipper.enable(:travel_pay_view_claim_details)
```

Comment:
Can you mock the `Flipper.enable` and `Flipper.disable` in this file? CI tests run in parallel so sometimes there are intermittent failures when flippers are enabled or disabled directly. See the [example here](https://depo-platform-documentation.scrollhelp.site/developer-docs/feature-toggles-guide#FeatureTogglesGuide-Backendexample)
```ruby
      # Recommended to limit the feature flag state to the scope of this test
      allow(Flipper).to receive(:enabled?).with(:feature_flag).and_return(true)
      # Not recommended. Affects global state and can have unintended side effects when tests run in parallel
      Flipper.enable(:feature_flag)
```

Reply:
Interesting, I didn't realize. Thanks!